JWBestPracticeApps/AutoplayVideoFeed/AutoplayVideoFeedViewController.swift

```diff
@@ -28,4 +29,22 @@ class AutoplayVideoFeedViewController: FeedTableViewController {
         }
         return nil
     }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: FeedItemCellIdentifier, for: indexPath) as! FeedItemCell
+        
+        // Add player view to the container view of the cell
+        cell.player = feed[indexPath.row]
+        
+        // Search for the title on the thumbnails Array and add the corresponding image
+        if let playerTitle = cell.player?.config.title {
```

Comment:
why not use  cell.play.config.image ?

Reply:
Oh that's because I added an UIImageView on top of the player to display the thumbnail until the player starts. For this I have added the thumbnail images on the Assets and using the name of the asset to create the UIImage

Reply:
no i mean why not set the thumbnail in the config, instead of having this logic where you filter through a hashmap and get the image matching a title.

Reply:
you could get rid of `thumbnailsIdentifiers`. 
just use `cell.thumbnailImageView.image = UIImage(named: config.image)`